mis_builder/models/mis_builder.py

```diff
@@ -800,7 +936,21 @@ def display_settings(self):
     def compute(self):
         self.ensure_one()
 
-        aep = self.report_id._prepare_aep(self.root_account)
+        sub_report_ids = self.env.context.get('sub_report_ids')
+
+        if sub_report_ids:
+            report_id = self.env['mis.report'].browse(sub_report_ids)
```

Comment:
This isn't working.

Reply:
https://github.com/OCA/mis-builder/pull/153/commits/8afb3c68b4d6d4755b7a6c798602be89b69574ff